app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def googleAuth
+    # Gets access tokens from the google server
+    auth = request.env["omniauth.auth"]
+    user = User.find_or_create_by(google_uid: auth.uid)
+    user.first_name = auth.info.first_name
+    user.last_name = auth.info.last_name
+    user.google_token = auth.credentials.token
+    user.email = auth.info.email
```

Comment:
Let's revisit this later to refactor. Wondering if there's a cleaner way to create this user?